### PR TITLE
resolves : explain how users are expected to use the results and how this will be achieved. #14

### LINKED ISSUE #14: explain how users are expected to use the results and how this will be achieved.

### PR DESCRIPTION
**Pull request by JavierCladellas**
- Closes #14

### CODE REVIEW

**Comment by JavierCladellas** [comment]
@prudhomm Is there a specific procedure to follow for adding and compressing new images ? I executed the `compress.py` script. Is that the way?

**Comment by prudhomm** [comment]
the problem is that for the journal we have to provide high quality but on HAL high quality images are too big and upload fails. so we need to have both. 
I used compress.py to compress the images

File: article.hid2-ub-cicd.ppam24.tex [hunk 1]
```diff
@@ -124,6 +124,25 @@ \subsection{Final Analysis and Urban Scale Energy Evaluation}
...
 
+In addition to these exports, the simulator condenses, aggregates and saves relevant quantities on the fly to avoid having to manipulate large data volumes after the simulation. These metrics concern weather information, energy consumption and comfort assessment, and are exported in JSON format. The produced files are uploaded to a data management platform via the CI/CD pipeline, who is then in charge of notifying a different workflow in order to render insightful figures that showcase the simulation results.
+
+Insights are presented in the form of a comprehensive report and are automatically deployed on a website built using the \textit{Antora} framework. End users are finally able to explore the simulation outputs in a detailed but malleable manner, for both the city and the building scales.
+
+The figure \ref{fig:ktirio-cases-report} presents a gallery of city reports that are taken as test cases for the Urban Building pilot, as well as an example of a city report section showing the occlusion factor and heat flow plot.
```
Comment (prudhomm) [inline article.hid2-ub-cicd.ppam24.tex]: use \cref : `\Cref{...}` will write Figure xxx presents ...

Comment (prudhomm) [reply article.hid2-ub-cicd.ppam24.tex]: see #25

File: article.hid2-ub-cicd.ppam24.tex [hunk 2]
```diff
@@ -124,6 +124,25 @@ \subsection{Final Analysis and Urban Scale Energy Evaluation}
 
 The final step involves the \textit{UBEM Simulator}, which generates large-scale outputs that summarize the overall energy consumption and environmental impact of buildings on an urban scale. This comprehensive urban scale analysis merges data from the building energy and air quality models to provide a holistic view of urban environmental quality.
 
+In addition to these exports, the simulator condenses, aggregates and saves relevant quantities on the fly to avoid having to manipulate large data volumes after the simulation. These metrics concern weather information, energy consumption and comfort assessment, and are exported in JSON format. The produced files are uploaded to a data management platform via the CI/CD pipeline, who is then in charge of notifying a different workflow in order to render insightful figures that showcase the simulation results.
```
Comment (prudhomm) [inline article.hid2-ub-cicd.ppam24.tex]: add a \cref for the Fig 10 that shows the entire workflow including the comprehensive report at the end

Comment (prudhomm) [reply article.hid2-ub-cicd.ppam24.tex]: perhaps link to the section where we describe the HPCOps workflow and hence the corresponding Figure  10

**Comment by JavierCladellas** [comment]
> the problem is that for the journal we have to provide high quality but on HAL high quality images are too big and upload fails. so we need to have both. I used compress.py to compress the images

Could you check if the image quality in the paper suits the needs ?

**Review by prudhomm — CHANGES_REQUESTED** [review]
use cref, link to CI/CD HPCopps workflow and the generation of the report

**Review by prudhomm — APPROVED** [review]

**Pull request merged**